src/components/hub/elements/InformativeMessage.js

```diff
@@ -4,19 +4,22 @@ import { useLocation } from "react-router-dom";
 export default function InformativeMessage({
   text,
   params,
+  isUserExistInStore,
   isPrevMesssageUsers,
 }) {
   const { pathname, hash } = useLocation();
 
   return (
     <div
       className={
-        isPrevMesssageUsers
-          ? "informative-message mt-10"
-          : "informative-message"
+        "informative-message" +
+        (isUserExistInStore ? "" : "--disabled") +
+        (isPrevMesssageUsers ? " mt-10" : "")
       }
       onClick={() =>
-        addSuffix(pathname + hash, `/user?uid=${params?.user?._id}`)
+        isUserExistInStore
+          ? addSuffix(pathname + hash, `/user?uid=${params?.user?._id}`)
```

Comment:
instead of passing 'params' can we pass a `user_id` only?

Reply:
Also, we may get rid of `isUserExistInStore ` here and always allow to go to user profile. But then , on profile page, we can check whether users exists in redux. If not - load it from server